appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/repositories/DiscoveryAppsRepository.kt

```diff
@@ -0,0 +1,78 @@
+package org.kinecosystem.appsdiscovery.sender.repositories
+
+import android.content.Context
+import android.os.Handler
+import android.os.Looper
+import org.kinecosystem.appsdiscovery.sender.model.EcosystemApp
+import org.kinecosystem.appsdiscovery.sender.model.EcosystemAppResponse
+import org.kinecosystem.appsdiscovery.sender.model.hasNewData
+
+class DiscoveryAppsRepository(context: Context){
+
+    private var local = DiscoveryAppsLocal(context)
+    private var remote = DiscoveryAppsRemote()
```

Comment:
`local` and `remote` vars should be on ctr as param for unit testing usage.

Reply:
cool